File: generated-sources/scala-gatling/mojang-api/src/gatling/scala/com/github/asyncmc/mojang/api/scala/gatling/model/OrderStatisticsRequest.scala

package com.github.asyncmc.mojang.api.scala.gatling.model


case class OrderStatisticsRequest (
    _metricKeys: List[OrderStatistic]
)
object OrderStatisticsRequest {
    def toStringBody(var_metricKeys: Object) =
        s"""
        | {
        | "metricKeys":$var_metricKeys
        | }
        """.stripMargin
}

File: generated-sources/kotlin-spring/mojang-api/src/main/kotlin/com/github/asyncmc/mojang/api/kotlin/spring/model/OrderStatisticsRequest.kt
package com.github.asyncmc.mojang.api.kotlin.spring.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import com.github.asyncmc.mojang.api.kotlin.spring.model.OrderStatistic
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * The body of the request to get the order statistics
 * @param metricKeys 
 */
data class OrderStatisticsRequest (

        @get:NotNull 
        @JsonProperty("metricKeys") val metricKeys: List<OrderStatistic>
) {

}

